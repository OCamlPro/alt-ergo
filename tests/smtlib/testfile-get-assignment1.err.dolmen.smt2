(set-logic ALL)
; The next instruction should fail because we did not generate a model
(get-assignment)