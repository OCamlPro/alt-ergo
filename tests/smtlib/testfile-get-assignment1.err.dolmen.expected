(error "(get-assignment) requires model generation, but it is disabled (try --produce-models)")
